Jenkinsfile-Release

```
@@ -71,3 +69,30 @@ dockerizedBuildPipeline(
     buildNotifications(currentBuild, env, 'main')
   }
 )
+
+String verifyIndexYamlAndTarFile(String chart) {
+  // Get current version
+  def indexFile = readYaml file: 'docs/index.yaml'
+  String version = indexFile.entries[chart][0].version
+
+  sleep(60) // seconds
+
+  // Check tar file
+  String repo_url = 'https://sonatype.github.io/helm3-charts/'
+  verifyDownloadLinks(
+    urlParts: [repo_url, chart, '-', version],
+    urlSuffixes: ['.tgz']
```

You can add this after https://github.com/sonatype/jenkins-shared/pull/208 is merged:
```suggestion
    urlSuffixes: ['.tgz'], retryCount: 2, retryDelay: 60
```
Also remove the `sleep(60) // seconds` line above.